### ISSUE TITLE
Google Search redirection to spam protection ipv4.google.com

### ISSUE THREAD

**Issue by MarcGamesons** [EFForg/https-everywhere#13745]
Type: other

Hi,

since I started using httpseverywhere I get redirected to the spam protection site of google while using their search. The url I am getting redirected to is https://ipv4.google.com/sorry/index. I then have to solve a captcha, sometimes multiple after I just solved the previous one.

Is there any way for me to troubleshoot this, maybe with some networking tools like Wireshark? (So that I can find out if it really is httpseverywhere) Would be nice if someone could provide a tutorial.

Thanks in advance to everyone that is trying to help.

**Comment by jeremyn** [comment]
@MarcGamesons Thanks for your report. I've occasionally been redirected to Google's captcha page but I've never associated it with HTTPS Everywhere. Can you reproduce this consistently and if so, how?

**Comment by MarcGamesons** [comment]
@jeremyn The problem is that I only know that disabling the extension resolves the issue. I am not able to reproduce it in any way, I just search something on google and the spam protection pops up.

**Comment by jeremyn** [comment]
I don't know what triggers the Google spam catcher but I can't think of anything HTTPS Everywhere does that would trigger it. In fact HTTPS Everywhere should reduce the number of browser requests because it should rewrite URLs that would otherwise require a redirect response.

What operating system and browser are you using? If you recently started using Tor or a VPN, that might explain the spam warnings, though not why the warnings (apparently) decrease when you disable HTTPS Everywhere.

**Comment by MarcGamesons** [comment]
Sorry for the late reply,

I will close this issue for now because I am not able to consistently reproduce the error. It might be that something else is involved, maybe there is something configured in a wrong way, on my end. All I know is that it got worse since I started using HTTPS Everywhere and Privacy Badger.

I had these spam protection popups in the past (rarely) but not as frequent, I already did virus scans and they were negative. But with 4 people living at my home which all use their devices to occasionally google stuff maybe that is just to much and therefore triggering the spam protection. I don't know.

But thanks anyway for trying to help, I will still use both extensions because they are great and better than the extensions that I used before.

Have a nice day
